pkg/apis/virt/v1alpha1/host.go

```diff
@@ -36,7 +36,7 @@ type HostSpec struct {
 	// Certificate SHA-1 fingerprint, called thumbprint by VMware.
 	Thumbprint string `json:"thumbprint,omitempty"`
 	// Credentials.
-	Secret core.ObjectReference `json:"secret,omitempty"`
+	Secret *core.ObjectReference `json:"secret,omitempty"`
```

Comment:
Making a pointer makes the yaml easier in that the property can be omitted all together instead of:
```
secret: {}
```